### ISSUE TITLE
React hook missing dependency warning (react-hooks/exhaustive-deps) not appearing

### ISSUE THREAD

**Issue by sayanything830** [microsoft/vscode-eslint#852]
Running my lint script still notifies me of a missing dependency and up until 2 days ago (December 17th, 2019) there was a visual warning about missing dependencies. After latest extension update, I no longer have the visual warning in VSCode.

Reading the docs, it is not clear if this is a change that needs to be made with editor settings, or if this warning was intentionally removed.

Any further information would be appreciated.

```
// Example of warning from my terminal
warning  React Hook React.useCallback has a missing dependency: 'watchForErrors'. 
Either include it or remove the dependency array  
react-hooks/exhaustive-deps
```

**Comment by dbaeumer** [comment]
@sayanything830 where was the visual warning. On the ESLint status bar icon?

**Comment by sayanything830** [comment]
@dbaeumer The visual warning appeared on code as a yellow wavy line and provided an option to auto-fix dependencies.

**Comment by sayanything830** [comment]
Closing issue, warning is appearing with latest update.
